include/wx/dynlib.h

```diff
@@ -251,10 +251,13 @@ class WXDLLIMPEXP_BASE wxDynamicLibrary
     wxDllType Detach() { wxDllType h = m_handle; m_handle = NULL; return h; }
 
     // unload the given library handle (presumably returned by Detach() before)
-    static void Unload(wxDllType handle);
+    static bool Unload(wxDllType handle, wxString* errorDest = NULL);
 
     // unload the library, also done automatically in dtor
-    void Unload() { if ( IsLoaded() ) { Unload(m_handle); m_handle = NULL; } }
+    bool Unload();
+
+    // return error message related to the latest failure
+    wxString GetErrorStr() const { return m_lastError; }
```

Comment:
If we keep the function, I'd also rename it to either `GetError()` or `GetErrorMessage()`, both of which are already used in the API, while `GetErrorStr()` isn't.